src/components/application_manager/include/application_manager/app_launch/app_launch_data_json.h

```diff
@@ -132,12 +126,12 @@ class AppLaunchDataJson : public AppLaunchDataImpl {
   /**
    * @return pointer to AppLaunch data block in Json file
    */
-  Json::Value& GetSavedApplicationDataList() const;
+  Json::Value& GetSavedApplicationDataList(Json::Value& dictionary) const;
```

Comment:
@YarikMamykin missing param description

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/3dcb90541c1ee0310a396d2564a95eba3e12237e